index.html

```diff
@@ -68,23 +63,30 @@ <h2 class="mx-1rem">Question:</h2>
             quaerat asperiores, eius alias doloribus ut doloremque laudantium
             soluta ipsum enim!
           </p>
-          <button class="card__toggle-answer-button">Show Answer</button>
+          <button class="card__toggle-answer-button" data-js="toggle-answer">
+            Show Answer
+          </button>
+          <h2
+            class="card__answer-heading hidden mx-1rem"
+            data-js="answer-heading"
+          >
+            Answer:
+          </h2>
+          <p class="card__answer-text hidden mx-1rem" data-js="answer">
+            The answer is 42.
```

Comment:
haha nice! its alsways 42!

Reply:
"You're really not going to like it," observed Deep Thought.
"Tell us!"
"All right," said Deep Thought. "The Answer to the Great Question..."
"Yes..!"
"Of Life, the Universe and Everything..." said Deep Thought.
"Yes...!"
"Is..." said Deep Thought, and paused.
"Yes...!"
"Is..."
"Yes...!!!...?"
"Forty-two," said Deep Thought, with infinite majesty and calm.”